internal/app/service.go

```diff
@@ -163,6 +163,11 @@ func (svc *Service) MakeItRun() error {
 
 	svc.ctx.stop = stop
 
+	httpErrors := make(chan error)
+	defer close(httpErrors)
+
+	svc.webserver.StartWebServer(httpErrors)
```

Comment:
Please verify that this works when Trigger is set to `http`. i.e the route will be added after starting the webserver, which should be fine, but needs to be tested.